app/models/action.rb

```diff
@@ -1,6 +1,10 @@
 class Action < ActiveRecord::Base
   belongs_to :page
   belongs_to :action_user
+  after_create :update_page_action_count
 
+  def update_page_action_count
+    self.page.action_count = self.page.actions.count
```

Comment:
Another fine place to use `increment!`.
Don't use `self` when it's not required. Only necessary when calling a write accessor.

Also, I don't think it's wise to call `page.actions.count` on `create`. That's quite a performance hit, especially on the more popular pages. Why not just increment by 1?